cmake/test.cmake

```diff
@@ -74,6 +75,13 @@ function(mayaUsd_add_test test_name)
             "used with PYTHON_MODULE or PYTHON_COMMAND")
     endif()
 
+    # set the working_dir
+    if(PREFIX_WORKING_DIRECTORY)
+        set(working_dir ${PREFIX_WORKING_DIRECTORY})
+    else()
+        set(working_dir ${CMAKE_CURRENT_SOURCE_DIR})
+    endif()
+
```

Comment:
Now that I look at this again, I realize this can simplified. I will make a commit to address this.
if(NOT PREFIX_WORKING_DIRECTORY)
      set(working_dir ${CMAKE_CURRENT_SOURCE_DIR})
endif()

Reply:
Hmm... I don't quite understand your fix - I think the `set(working_dir ${PREFIX_WORKING_DIRECTORY})` is needed, isn't it?   Or are there other changes, too? I guess I'll wait for the actual commit.

Reply:
@elrond79  Sorry, Yes ignore my comment here. I am not sure what I was thinking when I wrote this comment. It has been a long day...... 

Everything as is should be fine. Thanks.